crowdsec-docs/docs/cti_api/integration_maltego.md

```diff
@@ -0,0 +1,141 @@
+---
+id: integration_maltego
+title: Maltego Transforms
+sidebar_position: 4
+---
+
+Maltego transforms, which allow users to enrich IP entities in maltego with CrowdSec CTI intelligence.
+
+# Deployment Guide
```

Comment:
It's strange, but the dynamic side table of content seems to only display from H2 and down.
Can you add a # to all your section titles in order to see Deployment and User Guide in the table of content ?